### ISSUE TITLE
Implement funders page

### ISSUE THREAD

**Issue by uniqueg** [elixir-cloud-aai/landing-page#66]
The `/funding` route is currently implement as a blank page (see https://elixir-cloud.dcc.sib.swiss/funding).

We have now created a Notion page/table with one funding instrument per row. This table should be used to implement the funding page, showing one card per funding instrument/row, quite similar to, e.g., the `/partners` page: https://elixir-cloud.dcc.sib.swiss/partners

The table has the following properties:

| Name | Notion type |
| --- | --- |
| Instrument | Title |
| ProjectTitle | Text |
| Recipients | Multi-select |
| Duration | Date (includes end date!) |
| Website | URL |
| Icon | URL |
| IconDark | URL |

Here is a mock up of what the card could look like:
![image](https://user-images.githubusercontent.com/10855418/210151425-b40ab585-d894-4dce-99e4-930c176f5713.png)

And here is an example row of the Notion table:
![image](https://user-images.githubusercontent.com/10855418/210151448-81c56510-5e5d-480b-8706-ea438c93d0ea.png)

The following behaviors should be implemented:
- A search bar (just like on the `/partners` page: https://elixir-cloud.dcc.sib.swiss/partners)
- Sort order by
   1. end date (from `Duration`); descending
   2. start date (from `Duration`); ascending
   3. `Instrument`; ascending
   4. `ProjectTitle`; ascending
   In other words, ongoing or most recent funding should be on the top, with the ones extending most into the future appearing first (hence descending order of end date); in case of equal end dates, longer running funding instruments should appear before shorter ones (hence _ascending_ order of start date); if there are still ties, instruments should then be alphabetically sorted by instrument name first, and project title second
- Clicking on a card should lead to the `Website`
- Above the search box and the cards, there should be a clearly visible banner or similar saying "We would like to express our gratitude to our funders" (or similar)

**Comment by git-anurag-hub** [comment]
Great work @tanya17-05, can you please change the fonts/formats for the subheadings according to the ones in the contributor's screen.

**Comment by tanya17-05** [comment]
Sure @git-anurag-hub
